_data/banking.yml

```diff
@@ -288,9 +288,12 @@ websites:
     - name: Commonwealth Bank of Australia
       url: https://www.commbank.com.au/
       img: commbank.png
-      tfa: No
-      twitter: CommBank
-      facebook: commonwealthbank
+      tfa: Yes
+      sms: Yes
+      software: Yes
+      phone: No
```

Comment:
Please remove the `phone: No` part as we don't list any tags with the value "No".